{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -20,12 +20,12 @@
 import sys
 from subprocess import check_call
 
-import edx_theme
-
 import django
 from django.conf import settings
 from django.utils import six
 
+import edx_theme
```

Comment:
Hmm, I would have thought so, but isort keeps putting it there. I tried pushing up the change but it didn't help. I wish that isort would give me more details in the error message as to why it thinks the imports are in the wrong order.